charts/kueue/templates/crd/kueue.x-k8s.io_clusterqueues.yaml

```diff
@@ -177,31 +177,36 @@ spec:
                     before borrowing or preempting in the flavor being evaluated.
                   properties:
                     whenCanBorrow:
-                      default: Borrow
+                      default: ReadyToUse
                       description: |-
                         whenCanBorrow determines whether a workload should try the next flavor
                         before borrowing in current flavor. The possible values are:
 
-                        - `Borrow` (default): allocate in current flavor if borrowing
+                        - `ReadyToUse` (default): allocate in the current flavor if borrowing
                           is possible.
                         - `TryNextFlavor`: try next flavor even if the current
                           flavor has enough resources to borrow.
+                        - `Borrow` (deprecated): old name for `ReadyToUse`; please use new name.
                       enum:
-                        - Borrow
+                        - ReadyToUse
                         - TryNextFlavor
+                        - Borrow
                       type: string
                     whenCanPreempt:
                       default: TryNextFlavor
                       description: |-
                         whenCanPreempt determines whether a workload should try the next flavor
                         before borrowing in current flavor. The possible values are:
 
-                        - `Preempt`: allocate in current flavor if it's possible to preempt some workloads.
+                        - `ReadyToUse`: stop looking for more flavors once Preempting flavor is
+                          found and it's possible to preempt some workloads.
```

Comment:
Nit: again assuming this is correct https://github.com/kubernetes-sigs/kueue/issues/6187#issuecomment-3363305862

`ReadyToUse`: stop looking for more flavors once Preempting flavor is found and it's possible to preempt some workloads.
-> 
`ReadyToUse`: use current flavor if preemption allows workload to fit in the current flavor